app/src/main/java/org/fossasia/phimpme/editor/fragment/MainMenuFragment.java

```diff
@@ -99,27 +100,37 @@ public void onClick(View v) {
     public void highLightSelectedOption(int mode) {
 
         menu_filter.setBackgroundColor(Color.TRANSPARENT);
+        menu_filter.setAlpha(1f);
         menu_enhance.setBackgroundColor(Color.TRANSPARENT);
+        menu_enhance.setAlpha(1f);
         menu_adjust.setBackgroundColor(Color.TRANSPARENT);
+        menu_adjust.setAlpha(1f);
         menu_stickers.setBackgroundColor(Color.TRANSPARENT);
+        menu_stickers.setAlpha(1f);
         menu_write.setBackgroundColor(Color.TRANSPARENT);
+        menu_write.setAlpha(1f);
 
-        int color = ContextCompat.getColor(context, R.color.md_grey_200);
+        int color = ContextCompat.getColor(context, R.color.grey);
```

Comment:
Highligting is already present but with very light grey color.
There is no need to make it dark grey and add alpha to it.